CITATION.cff

```diff
@@ -0,0 +1,75 @@
+################################# MIT LICENSE ##################################
+#
+# MIT License
+#
+# Copyright (c) 2022 Pablo Hamacher
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+################################################################################
```

Comment:
See my comment in the `.github/workflows/cffconvert.yml` file.

Reply:
GPL 2.0 is way different from MIT.  Without such a header in GPL 2.0, the file would not meet the requirements of the license.  I can remove them, if you want to, but there are also numerous other projects which do have an MIT license header.

In the terms of art, constants like these give the content a frame and have a major effect on the aesthetic recognition of a source file.  Beautiful source files are known to produce very few bugs.  And really, this is absolutely true.

Reply:
I suppose that those projects are also all a lot bigger than this project. 😅 

That sounds like a bold claim, do you have any support for that?

Reply:
The header was removed.

Reply:
You asked for it:  https://github.com/godotengine/godot.

Reply:
Yeah, but godot is definitely a bigger project than this 😄